coordinator/tests/validators/request-validator.spec.ts

```diff
@@ -199,9 +199,9 @@ test("Should reject message where MedicationRequests have different authoredOn",
     medicationRequests[0].authoredOn = "2020-01-01T00:00:00.000Z"
     expect(validator.verifyPrescriptionBundle(bundle, false)).toEqual([{
         "apiErrorCode": "INVALID_VALUE",
-        "message": "Expected all MedicationRequests to have the same value for authoredOn. Received \"2020-01-01T00:00:00.000Z\",\"2020-01-02T00:00:00.000Z\".",
```

Comment:
does this test still pass with the removal of `\` ?

Reply:
It does. The two strings are identical when you compare them:
```
"use strict"

'Expected all MedicationRequests to have the same value for authoredOn. Received "2020-01-01T00:00:00.000Z","2020-01-02T00:00:00.000Z".' === "Expected all MedicationRequests to have the same value for authoredOn. Received \"2020-01-01T00:00:00.000Z\",\"2020-01-02T00:00:00.000Z\"."
```
Single quotes does all the escaping for you

Reply:
ahh of course thats my bad